sdk/hdinsight/Azure.ResourceManager.HDInsight.Containers/src/autorest.md

```diff
@@ -0,0 +1,14 @@
+# Generated code configuration
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+``` yaml
+
+azure-arm: true
+require: https://github.com/Azure/azure-rest-api-specs/blob/5372f410f6af3de9559b63defbd556a7b10c4e65/specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/readme.md
+library-name: HDInsightContainers
+namespace: Azure.ResourceManager.HDInsight.Containers
+clear-output-folder: true
+skip-csproj: true
+output-folder: Generated/
```

Comment:
Could you please also include the content of `autorest.md` in the template starting from line 19 to the end of the template file?
These contents are required to generate the SDKs.
Here is the template: https://github.com/Azure/azure-sdk-for-net/blob/2c1d884cc72b7af6ef1606396802a8059a687dee/eng/templates/Azure.ResourceManager.Template/src/autorest.md?plain=1#L19

Reply:
Hi @ArcturusZhang , 
I respect your comment and did some test work after received this comment.

However, I found one error if we apply the 
format-by-name-rules:  *Uri': 'Uri'.  The impact of applying this is the SDK can't work due to the Uri end with "/" , but our RP doesn't support.

For this time, I hope that we can skip this. Thanks.

Reply:
Hi @ArcturusZhang , I applied the rules excepts for the Uri part, thanks